src/Aquifer.Tiptap/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "aquifer.typescript",
+    "version": "1.0.0",
+    "main": "app.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "tsc --build",
+        "clean": "tsc --build --clean"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "description": "",
+    "dependencies": {
+        "aquifer-tiptap": "https://github.com/BiblioNexusStudio/aquifer-tiptap#1.0.9"
```

Comment:
We'll need to bump this whenever we create a new tag for aquifer-tiptap.  I added a note in the readme about this.